nrich-notification/src/main/java/net/croz/nrich/notification/constant/NotificationConstants.java

```diff
@@ -47,6 +49,8 @@ public final class NotificationConstants {
 
     public static final String VALIDATION_FAILED_CONTENT_CODE = "notification.validation-failed.content";
 
+    public static final String VALIDATION_FAILED__CODE = "notification.validation-failed";
```

Comment:
We have two underscores here. I'm not sure if that is intentional?

Reply:
No it isn't, fixed